src/runtime/Makefile

```diff
@@ -112,6 +113,9 @@ KERNELDIR := $(PKGDATADIR)
 IMAGEPATH := $(PKGDATADIR)/$(IMAGENAME)
 FIRMWAREPATH :=
 FIRMWAREVOLUMEPATH :=
+TDVFFIRMWAREPATH := $(PREFIXDEPS)/share/tdvf/OVMF.fd
+TDVFFIRMWAREVOLUMEPATH := $(PREFIXDEPS)/share/tdvf/OVMF_VARS.fd
+TDXKERNELPARAMS := force_tdx_guest tdx_disable_filter
```

Comment:
Shouldn't the kata machinery append the kernel parameters "auto-magically" in case of  ` confidential_guest = true`?

Reply:
Not really, @wainersm.  For now these parameters are needed, but we're not sure if that will be the case with newer versions of the kernel, neither what's the situation with AMD SEV.

Reply:
There's also another thing, @wainersm, we're currently hardcoding the `confidential_guest` value depending on the configuration file used, so we can generate everything as part of the very same build.  Adding logic to do "smarter" things at this point of the project could actually lead us to more complexity without much gain.

We should, though,revisit this discussion when merging things into the `main` branch.

Reply:
got it @fidencio and I agree.